### ISSUE TITLE
Look at Docusaurus

### ISSUE THREAD

**Issue by davidhund** [davidhund/styleguide-generators#57]
https://docusaurus.io/

**Comment by ran-dall** [comment]
@davidhund 👉 I feel like **Docusaurus** is meant for _general documentation_, as opposed to the focused and more elegant Pattern Generators currently on the list.

**Comment by davidhund** [comment]
@ran-dall You are right, it is. However, there is certainly some overlap and Docusaurus might be a 'Quick and Dirty' tool for a Pattern Library.

Maybe we should introduce another section for 'similar/related' tools?

**Comment by ran-dall** [comment]
> However, there is certainly some overlap and Docusaurus might be a 'Quick and Dirty' tool for a Pattern Library.

@davidhund It's true, as it does allow you to _build React components in your documentation_; but not much else. If we're focusing on Pattern Generators; and not Style Guide Generators (cause there is some confusion there); then I'd argue this wouldn't be a Pattern Generator. I'd say the difference between the two is the lack of process in a Style Guide Generator (i.e., much less opinionated (e.g., Pattern Labs vs. Aigis).

> Maybe we should introduce another section for 'similar/related' tools?

@davidhund I was getting to this on my last point, but I think it would be wise to point out the difference between a "Pattern Library Generator" and a "Style Guide Generator;" before we implement a "Similar/Related" area (that can be ambiguous and add some confusion). While similar and have overlap; they are not the same thing, IMO.

I would propose maybe adding a lightweight explanation (and resources) of the two methods and re-categorizing the list at the top level so that it's clear what tool follows which of the two. What would be your thoughts?
